splunk_connect_for_snmp/inventory/tasks.py

```diff
@@ -106,9 +128,18 @@ def generate_poll_task_definition(
     return task_config
 
 
+def add_profile_to_assigned_list(
+    assigned_profiles: dict[int, list[str]], frequency, profile_name
```

Comment:
as frequency seems to be an int, I am wondering whether  it should not be better to use type annotation and frequency: int